packages/synapse-react-client/src/components/RecentPublicationsGrid/RecentPublicationsGrid.tsx

```diff
@@ -64,104 +73,122 @@ function RecentPublicationsGrid(props: RecentPublicationsGridProps) {
-                      key={index}
-                      variant="overline"
-                      sx={{
-                        backgroundColor: 'grey.300',
-                        borderRadius: '3px',
-                        padding: '4px 8px',
-                        border: 'none',
-                        lineHeight: 'initial',
-                      }}
-                    >
-                      {tag}
-                    </Typography>
-                  ),
-                )}
-            </Box>
+          <Box>
+            {pub.values[categoryColIndex] && (
+              <Typography
+                variant="overline"
+                fontSize={'14px'}
+                sx={{
+                  backgroundColor: 'grey.300',
+                  borderRadius: '3px',
+                  padding: '4px 8px',
+                  border: 'none',
+                  lineHeight: 'initial',
+                }}
+              >
+                {pub.values[categoryColIndex]}
+              </Typography>
+            )}
             <Typography
               variant="headline2"
               color="grey.1000"
+              fontSize={'21px'}
               sx={{
                 padding: '20px 0px',
               }}
             >
               {pub.values[titleColIndex]}
             </Typography>
-            <Typography
-              variant="body1"
-              color="grey.700"
-              sx={{
-                fontStyle: 'italic',
-                fontSize: '14px',
-              }}
-            >
-              {pub.values[journalColIndex]}
-            </Typography>
-            <Typography
-              variant="body1"
-              color="grey.700"
-              sx={{ fontSize: '14px' }}
-            >
-              {pub.values[publicationDateColIndex] &&
-                formatDate(
-                  dayjs(Number(pub.values[publicationDateColIndex])),
-                  'MMMM, YYYY',
-                )}
-            </Typography>
-          </CardContent>
+            <Box display={'flex'} gap={'8px'} flexDirection={'column'}>
+              <Typography
+                variant="body1"
+                color="grey.700"
+                fontSize={'14px'}
+                fontStyle={'italic'}
+                lineHeight={1.35}
+              >
+                {pub.values[journalColIndex]}
+              </Typography>
+              <Typography
+                variant="body1"
+                color="grey.700"
+                fontSize={'14px'}
+                lineHeight={1.35}
+                paddingBottom={{ xs: 0, md: '35px' }}
+              >
+                {pub.values[publicationDateColIndex] &&
+                  formatDate(
+                    dayjs(Number(pub.values[publicationDateColIndex])),
+                    'MMMM, YYYY',
+                  )}
+              </Typography>
+            </Box>
+          </Box>
         )}
       </Box>
     </Grid>
   )
 
   return (
-    <Box
+    <Grid
```

Comment:
Why `Grid` component if you're manually setting `display: grid`? I think if you're manually setting up a CSS Grid layout, you should probably stick to `Box`. The `Grid` components should be used if you're using the built-in Grid layout, but if you're overriding it, I think it is just more confusing.